src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
+  getArea(): number,
 }
 
 export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Resulting value should be bigger than 0');
+    }
+
+    if (this.a + this.b <= this.c) {
```

Comment:
There is a possible mistake. How do you know that hypotenuse always will be the last parameter? Recommend using Math.max to find the longest side.

Reply:
If I correctly remmember in algebra the third side by name `c` its allways hypotenuse, so it's allways has longest side, so it wasn't misstake.